### PR TITLE
fix pan-os-list-applications command bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
when running ```!pan-os-list-applications``` on panorama >= version 9 instance using panorama integration, the !pan-os-list-applications command fails on AttributeError.

## Screenshots
image

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed ut - https://app.circleci.com/pipelines/github/demisto/content/160552/workflows/18ae7924-4cd7-44c6-b13d-9046c9460eca/jobs/512598

File: Packs/PAN-OS/ReleaseNotes/1_12_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- fixed an issue where the command **pan-os-list-applications** failed when running on panorama instance that its version is nine and above.
```
Comment (bakatzir) [inline Packs/PAN-OS/ReleaseNotes/1_12_1.md:4]:
```suggestion
- Fixed an issue where the **pan-os-list-applications** command failed when running on a PAN-OS instance that its version is 9 or higher.
```

File: Packs/PAN-OS/ReleaseNotes/1_12_1.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Fixed an issue where the **pan-os-list-applications** command failed when running on a PAN-OS instance that its version is 9 or higher.
```
Comment (dantavori) [inline Packs/PAN-OS/ReleaseNotes/1_12_1.md:4]:
```suggestion
- Fixed an issue where the ***pan-os-list-applications*** command failed when running on a PAN-OS instance that its version is 9 or higher.
```

**Review by dantavori — APPROVED** [review]
nice

**Pull request merged**